src/lib/components/TabCapsule.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+	import { cn } from '$lib/utils';
+	import { faPlus } from '@fortawesome/free-solid-svg-icons';
+	import Fa from 'svelte-fa';
+
+	export let active: boolean = false;
+	export let label: string;
+
+	let isHovered = false;
```

Comment:
I think the component doesn't have `hovered` state can you remove this kub ?

Reply:
![Mov to Gif conversion](https://github.com/user-attachments/assets/12001937-9745-4e1a-b843-d630c6f8c820)
do you mean this isn't required, just change state when click right?

Reply:
Yep

Reply:
Ok ngub